pym/portage/dbapi/bintree.py

```diff
@@ -672,17 +693,17 @@ def _populate(self, getbinpkgs=False, getbinpkg_refresh=True):
 					multi_instance = False
 					invalid_name = False
 					build_id = None
-					if myfile.endswith(".xpak"):
+					if myfile.endswith("." + self.suffix):
```

Comment:
@zmedico If we use xpak as a general suffix, this will treat all packages as multi_instance. I assume this needs to be changed, then?